manage.py

```diff
@@ -777,6 +777,9 @@ def pre_commit():
     # Update examples/geojson/nodes.geojson and examples/geojson/spans.geojson
     subprocess.run(['libcoveofds', 'jsontogeojson', 'examples/json/network-package.json', 'examples/geojson/nodes.geojson', 'examples/geojson/spans.geojson'])
 
+    # Run mdformat
```

Comment:
```suggestion
    # Format Markdown files
```

Reply:
Also, please add to the docstring for this function.

Reply:
Done by editing first commit.